sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -868,6 +915,30 @@ private void setDefaultTransmissionTarget(String transmissionTargetToken) {
         }
     }
 
+    /**
+     * Set transmission interval. The interval should be between 3 seconds and 86400 seconds (1 day).
+     * Should be called before the service is started.
+     *
+     * @param seconds the latency of sending events to Analytics.
+     * @return <code>true</code> if the interval is set, <code>false</code> otherwise.
+     */
+    private boolean setInstanceTransmissionInterval(int seconds) {
+        if (mChannel != null) {
+            AppCenterLog.error(LOG_TAG, "Transmission interval should be set before the service is started.");
+            return false;
+        }
+        if (seconds < MINIMUM_TRANSMISSION_INTERVAL_IN_SECONDS || seconds > MAXIMUM_TRANSMISSION_INTERVAL_IN_SECONDS) {
+            AppCenterLog.error(LOG_TAG, String.format(Locale.getDefault(),
```

Comment:
This error message is overwritten during previous merge. There is @annakocheshkova's  comment [here](https://github.com/microsoft/appcenter-sdk-android/pull/1145/commits/bdd7d1ad682f04e16600c88705fbc139ad8276a1#r283722596).

We don't need to provide those in the error log.
Please revert this one.
```
AppCenterLog.error(LOG_TAG, "The transmission interval is not valid.");
```

Reply:
There was a confusion here, the original comment was about hardcoding constants in the message, the string.format solution is fine.